packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2079,42 +2213,15 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   private _getInitialVOIRange(image: IImage) {
-    if (this.voiRange && this.voiUpdatedWithSetProperties) {
-      return this.voiRange;
-    }
-    const { windowCenter, windowWidth } = image;
-
-    let voiRange = this._getVOIRangeFromWindowLevel(windowWidth, windowCenter);
-
-    // Get the range for the PT since if it is prescaled
-    // we set a default range of 0-5
-    voiRange = this._getPTPreScaledRange() || voiRange;
-
-    return voiRange;
-  }
-
-  private _getPTPreScaledRange() {
-    if (!this._isCurrentImagePTPrescaled()) {
-      return undefined;
-    }
-
-    return this._getDefaultPTPrescaledVOIRange();
-  }
-
-  private _isCurrentImagePTPrescaled() {
-    if (this.modality !== 'PT' || !this.csImage.isPreScaled) {
-      return false;
-    }
-
-    if (!this.csImage.preScale?.scalingParameters?.suvbw) {
-      return false;
+    if (
+      this.globalDefaultProperties?.voiRange &&
+      this.voiUpdatedWithSetProperties
+    ) {
+      return this.globalDefaultProperties.voiRange;
+    } else {
+      const { windowCenter, windowWidth } = image;
+      return this._getVOIRangeFromWindowLevel(windowWidth, windowCenter);
```

Comment:
why are you removing the _isCurrentImagePTPrescaled and _getPTPreScaledRange

Reply:
We've come to think that the viewport should't know if it's PT prescaled or something. We're thinking that it should't have this logical.

Reply:
Yeah you might be right, but let's deal with that in another PR and not this one